packages/core/src/components/table/table.stories.js

```diff
@@ -0,0 +1,351 @@
+      </colgroup>
+      <tr class="hds-table__header-row">
+        <th></th>
+        <th scope="col">8-12</th>
+        <th scope="col">12-14</th>
+        <th scope="col">14-16</th>
+        <th scope="col">16-18</th>
+      </tr>
+      <tbody class="hds-table__content hds-table__content--text-align-td-right">
+        ${unitsContent()}
+      </tbody>
+    </table>
+  </div>
+`;
+
+export const CustomHeaderBackgroundColorForDarkVariant = () => `
+  <style type="text/css">
+    .custom-background-color-1 {
+      --background-color: var(--color-tram);
+    }
+  </style>
+  <div class="hds-table-container">
+    <table class="hds-table hds-table--dark custom-background-color-1" aria-label="People">
+      <tr class="hds-table__header-row">
+        <th scope="col">First name</th>
+        <th scope="col">Surname</th>
+        <th scope="col">Age</th>
+        <th scope="col">Profession</th>
+      </tr>
+      <tbody class="hds-table__content">
+        ${tableRow('Lauri', 'Kekkonen', '39', 'Engineer')}
+        ${tableRow('Maria', 'Sarasoja', '62', 'Designer')}
+        ${tableRow('Anneli', 'Routa', '50', 'Meteorologist')}
+        ${tableRow('Osku', 'Rausku', '18', 'Mail carrier')}
+      </tbody>
+    </table>
+   </div>
+`;
+
+export const CustomHeaderBackgroundColorForLightVariant = () => `
+  <style type="text/css">
+    .custom-background-color-2 {
+      --background-color: var(--color-suomenlinna);
+    }
+  </style>
+  <div class="hds-table-container">
+    <table class="hds-table hds-table--light custom-background-color-2" aria-label="People">
+      <tr class="hds-table__header-row">
+        <th scope="col">First name</th>
+        <th scope="col">Surname</th>
+        <th scope="col">Age</th>
+        <th scope="col">Profession</th>
+      </tr>
+      <tbody class="hds-table__content">
+        ${tableRow('Lauri', 'Kekkonen', '39', 'Engineer')}
+        ${tableRow('Maria', 'Sarasoja', '62', 'Designer')}
+        ${tableRow('Anneli', 'Routa', '50', 'Meteorologist')}
+        ${tableRow('Osku', 'Rausku', '18', 'Mail carrier')}
+      </tbody>
+    </table>
+   </div>
+`;
+
+const wideTableContent = () => {
+  let content = "";
+
+  workTrial.default.forEach(work => {
+    content = content + `
+        <tr>
+          <td class="hds-table--text-align-right">${work['Paikka-ID']}</td>
+          <td>${work['Paikan tyyppi']}</td>
+          <td>${work['Tehtävänimike']}</td>
+          <td>${work['Ilmoitus vanhenee']}</td>
+          <td>${work['Toimiala/liikelaitos']}</td>
+          <td>${work['Työpaikka']}</td>
+          <td class="hds-table--text-align-right">${work['Postinumero']}</td>
+          <td class="hds-table--text-align-right">${work['Paikkoja']}</td>
+          <td class="hds-table--text-align-right">${work['Haastatteluun halutaan']}</td>
+        </tr>`
+  })
+  return content;
+}
+
+export const WideAndLongExample = () => `
+  <div class="hds-table-container" style="height: 600px;" tabindex="0">
```

Comment:
I'm not sure should we even try to fix focus behavior in the core package because the table might fit into screen in some cases 🤔 
On the other hand, is it bad if a large table without scrollbars does have this behavior?

Reply:
Well, in this case the table won't fit into the container, as there's only 600px height for it. But this is a good point. Not sure if we should include `tabindex="0"` to every table, since in mobile view they won't fit to the container?

Reply:
We discussed this on todays a11y meeting, and decided to set all hds tables focusable, to guarantee a11y. 

- [ ] Set all tables focusable